ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -337,7 +337,7 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
       `com.walmartlabs.ern:react-native:${versions.reactNativeAarVersion}`,
     );
     androidDependencies.regular.push(
-      `com.android.support:appcompat-v7:${versions.supportLibraryVersion}`,
+      `androidx.appcompat:appcompat:${versions.androidxAppcompactVersion}`,
```

Comment:
`com.walmartlabs.ern:react-native` already comes with a compile time dependency on `androidx.appcompat:appcompat`.

With the removal of the support library, this statement might not be necessary anymore at all. Have you tried what happens if `androidx.appcompat:appcompat` is not declared explicitly?

Reply:
You were right. Tested this without adding androidx dependency and it still works. Removing it.